lib/ansible/cli/galaxy.py

```diff
@@ -277,7 +277,7 @@ def init_parser(self):
         collections_path = opt_help.ArgumentParser(add_help=False)
         collections_path.add_argument('-p', '--collections-path', dest='collections_path', type=opt_help.unfrack_path(pathsep=True),
                                       action=opt_help.PrependListAction,
-                                      help="One or more directories to search for collections in addition "
+                                      help="One or more directories to search for collections (or install to) in addition "
```

Comment:
This collections_path option is not the one used by install (https://github.com/ansible/ansible/blob/devel/lib/ansible/cli/galaxy.py#L512-L514), and can include paths ansible-galaxy collection install will not use.

Reply:
Correct that, in `devel` branch, this `--collections-path` option is not used by `ansible-galaxy install`, and I'm not really trying to change that. For my own selfish purposes, I just want to consolidate our use cases such that we use the environment variables ANSIBLE_COLLECTIONS_PATHS and ANSIBLE_ROLES_PATH. And the reason I made this PR is because I found that non-obvious.

https://github.com/ansible/awx/blob/3624fe2cac7eccf845e46afbb88b6d2d8af7b41a/awx/playbooks/project_update.yml#L196-L197

You can use the `--collections-path` in _this_ install command, which is why I made this particular change to this line:

```
ansible-galaxy collection install awx.awx -p alan
```

My gripe was that this help text was written assuming the user was running `ansible-galaxy collection list -p alan/`, but the option was reused for the `ansible-galaxy collection install`.

Reply:
The `--collections-path` option is added in two locations and has different help texts (and different types and defaults). Changing the --help for the list subcommand here doesn't affect the option in `ansible-galaxy collection install --help`.